lasagne/layers/prelu.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import theano.tensor as T
+
+from .. import init
+from .. import nonlinearities
+
+from .base import Layer
+
+__all__ = [
+    "PReLULayer"
+]
+
+class PReLULayer(Layer):
+    """
+    A layer used to implement Parametric Rectified Linear Units (PReLU).
+    This layer simply applies the parametric nonlinearity to the previous
+    layer's output. 
+
+    Parameters
+    -----------
+    incoming : a :class:`Layer` instance or a tuple
+        The layer feeding into this layer, or the expected input shape.
+
+    alpha : double or None
+        The initial "leakiness" of the rectifier which is then learned.
+        If argument not provided, initial leakiness will be set to 0.25.
+
+    unique_chan_param : boolean or None
```

Comment:
@benanne That would certainly simplify the code, and if that's fine... that's fine. I just generally write towards how *I'd* use it. Agreed on name. I'll probably go with @JeffreyDF 's idea.